lib/compiler/src/cerl.erl

```diff
@@ -2063,6 +2063,7 @@ is_c_map(#c_literal{val = V}) when is_map(V) ->
 is_c_map(_) ->
     false.
 
+-doc "".
```

Comment:
This feels a bit strange to have this? Is the plan to fix them and add some documentation or leave them like this? 
Is `-doc "".` better than `-doc false.`?

Reply:
It is the same as the [current docs](https://www.erlang.org/doc/man/cerl#map_es-1). cerl is a bit of an odd beast as it is a (partially) documented, unsupported internal module.

Putting `-doc false` would mean that we would hide the function which is not the same as it having no documentation.